homeassistant/components/ads/__init__.py

```diff
@@ -149,6 +134,23 @@ def handle_write_data_by_name(call):
 class AdsHub:
     """Representation of an ADS connection."""
 
+    ADS_TYPEMAP = {
+        ADSTYPE_BOOL: pyads.PLCTYPE_BOOL,
+        ADSTYPE_BYTE: pyads.PLCTYPE_BYTE,
+        ADSTYPE_DINT: pyads.PLCTYPE_DINT,
+        ADSTYPE_INT: pyads.PLCTYPE_INT,
+        ADSTYPE_UDINT: pyads.PLCTYPE_UDINT,
+        ADSTYPE_UINT: pyads.PLCTYPE_UINT,
+    }
+
+    ADSError = pyads.ADSError
+    PLCTYPE_BOOL = pyads.PLCTYPE_BOOL
+    PLCTYPE_BYTE = pyads.PLCTYPE_BYTE
+    PLCTYPE_DINT = pyads.PLCTYPE_DINT
+    PLCTYPE_INT = pyads.PLCTYPE_INT
+    PLCTYPE_UDINT = pyads.PLCTYPE_UDINT
+    PLCTYPE_UINT = pyads.PLCTYPE_UINT
+
```

Comment:
I moved these inside the class as they were causing issues for mypy when it was inside `setup`.
I think this is correct.

Reply:
Constants should be moved out of here anyways IMHO...

However, make sure you actually are able to test such changes

Reply:
I was highlighting them in my self-reviews, to encourage owners to review. I guess it doesn't work here as there is no code owner!

Since I am unable to test the change, should I simply cancel this PR?

Reply:
> Since I am unable to test the change, should I simply cancel this PR?

Yes, it is not worth the risk of breaking stuff